lib/index.js

```diff
@@ -85,3 +95,40 @@ function send(track, fn){
     .send(payload)
     .end(fn);
 }
+
+/**
+ * Get a human readable name for the user:
+ * 1. Check traits.name
+ * 2. Check traits.firstName + traits.lastName
+ * 3. Check traits.firstName
+ * 4. Check traits.lastName
+ * 5. Check traits.username
+ * 6. Check traits.email
+ * 7. Use 'User ' + userId
+ * 8. Use 'Anonymous user ' + anonymousId
+ *
+ * @return {string}
+ */
+
+function getName(message) {
```

Comment:
we already have `msg.firstName()`, `msg.lastName()`, `msg.name()`, `msg.username()` that does alot of this lookup logic for you :)